### ISSUE TITLE
The default training steps is 50K？

### ISSUE THREAD

**Issue by YuKDseele** [lianqi1008/MCM#4]
I noticed that the settings in the main_compress.py are somewhat inconsistent with those described on arxiv,
in line 231，for epoch in range(last_epoch, args.epochs, 5):
so the epoch num is args.epoch/5 =20， then the model totally train 82783//32*20=51.74K steps . not the 250K.

and when I use the default λ(1e-4 ) during training, the bpploss in the validation decreased to 0.

**Comment by lianqi1008** [comment]
Hi, I've corrected the error in the code. The training runs for a total of 250K steps and we save checkpoints every 5 epochs. For your second question, could you provide the logs or screenshots for better clarity?

**Comment by YuKDseele** [comment]
When I used the provided training code for training, I found that the bpp_loss decreased significantly in the test phase of the first few epochs. Previous records were deleted, but now I have reproduced this situation. It can be observed that the bpp_loss is quite low around epoch 3, and the aux_loss has not converged by epoch 31. Is this due to the learning rate being too low for the cross-entropy loss? Below are the training configurations and two screenshots taken during training.
![20231103](https://github.com/lianqi1008/MCM/assets/114339354/e30b1a1a-57f1-4a5e-afd4-8a614a7a48fc)
<img width="1148" alt="202311032" src="https://github.com/lianqi1008/MCM/assets/114339354/121cfbb3-231c-405a-909a-31fbd1ea6375">
![202311033](https://github.com/lianqi1008/MCM/assets/114339354/84c62afd-0410-4cb3-8183-6c6554bceba1)

**Comment by lianqi1008** [comment]
Hi, I understand it. This may be due to the fact that joint R-D loss optimization for masked image reconstruction is a difficult task, my suggestion is to first set lambda to a large value ( for example 200 ) and train some epochs until the L1 loss drops below 0.1 (or lower), at that time the image can be reconstructed well. Afterward, you can adjust lambda to the required value and continue training.
